devices/tty-bios/tty-bios.c

```diff
@@ -188,9 +188,9 @@ static ttybios_t *ttybios_get(unsigned int minor)
 static ssize_t ttybios_read(unsigned int minor, addr_t offs, void *buff, size_t len, time_t timeout)
 {
 	ttybios_t *tty;
-	unsigned int l;
-	char sc, c;
 	time_t start;
+	char sc, c;
+	size_t l;
```

Comment:
Geeez, please don't use l as a variable name. It look exactly as 1... I was mistaken because of it on the below change (`buff + l`, which looks like `buff + 1`)

Reply:
Fixed, I'll try not to use 'l' as a variable name anymore.